lib/screens/transfer/send/send_confirmation/interactor/viewmodels/send_confirmation_commands.dart

```diff
@@ -28,7 +28,7 @@ class ShowTransferSuccess extends TransactionPageCommand {
   });
 }
 
-class ShownInvalidTransactionResaon extends TransactionPageCommand {
+class ShownInvalidTransactionReason extends TransactionPageCommand {
   final String resaon;
-  ShownInvalidTransactionResaon(this.resaon);
+  ShownInvalidTransactionReason(this.resaon);
```

Comment:
if you can please change resaon to reason (:

Reply:
But..  that is the change 😆

Reply:
Ohh I see the param too who write this event???? haha

Reply:
probably me lol

Reply:
Nah git lens says were Nick and Gery who worked on that file.

Typos fixed 👍